### PR TITLE
Flag to enable/disable remote signals (background job)

### PR DESCRIPTION
**Pull request by artsiomtserashkovich**
## Before submitting checklist

- [ ] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [ ] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

File: src/neptune/metadata_containers/run.py [hunk 1]
```diff
@@ -455,9 +459,10 @@ def _get_or_create_api_object(self) -> ApiExperiment:
     def _prepare_background_jobs(self) -> BackgroundJobList:
         background_jobs = [PingBackgroundJob()]
 
-        websockets_factory = self._backend.websockets_factory(self._project_api_object.id, self._id)
-        if websockets_factory:
-            background_jobs.append(WebsocketSignalsBackgroundJob(websockets_factory))
+        if self._enable_remote_signals:
+            websockets_factory = self._backend.websockets_factory(self._project_api_object.id, self._id)
+            if websockets_factory:
+                background_jobs.append(WebsocketSignalsBackgroundJob(websockets_factory))
```
Comment (aniezurawski) [inline src/neptune/metadata_containers/run.py:465]: Is this job also created for Model and ModelVersion?

Comment (artsiomtserashkovich) [reply src/neptune/metadata_containers/run.py:465]: Only for Run

Comment (artsiomtserashkovich) [reply src/neptune/metadata_containers/run.py:465]: FYI, I found another place where WS is used: https://github.com/neptune-ai/neptune-client/blob/fe600779c39b4a41b4935f4c4f69849e6b4abdcb/src/neptune/legacy/internal/execution/execution_context.py#L91, but based on what I see it's legacy part

**Review by aniezurawski — APPROVED** [review]

File: src/neptune/metadata_containers/run.py (hunk 1 above)
Comment (aniezurawski) [reply src/neptune/metadata_containers/run.py:465]: Don't bother.

**Pull request merged**